tests/test_prettytable.py

```diff
@@ -1220,9 +1234,9 @@ class TestStyle:
             pytest.param(
                 PLAIN_COLUMNS,
                 """
-Field 1        Field 2        Field 3        
-value 1         value2         value3        
-value 4         value5         value6        
+Field 1        Field 2        Field 3
+value 1         value2         value3
+value 4         value5         value6
```

Comment:
We need to keep these trailing spaces:
```suggestion
Field 1        Field 2        Field 3        
value 1         value2         value3        
value 4         value5         value6        
```